sonar-update-center-mojo/src/main/java/org/sonar/updatecenter/mojo/HttpDownloader.java

```diff
@@ -75,4 +78,24 @@ File downloadFile(URL fileURL, File toFile) {
     return toFile;
   }
 
+
+  boolean downloadUrlStillValid(URL fileURL) {
```

Comment:
I feel this naming is confusing, `download` sounds like the verb. May I suggest rather `checkUrlStillValid`

Reply:
and...

looking at the text of the log on line 83, it seems a good name would be `verifyDownloadUrl`